src/documents/InfusionEventSystem.md

```diff
@@ -162,26 +164,26 @@ In this case, you can use the long form record, where the listener record takes
   </thead>
   <tr>
     <td><code>listener</code>, <code>func/funcName</code></td>
-    <td><code>String</code></td>
+    <td><code>{String}</code></td>
     <td>This holds the actual designation of the function which is to be the listener, which was the string that consisted of the entire "short form" for the listener definition above. This, as before, holds either
     the name of a global function or an IoC reference to a function. The `func`/`funcName` form is supported for consistency with the syntax for [Invokers](Invokers.md) and the name `listener` will be deprecated for the Infusion 2.0 release.
```

Comment:
There are quite a number of occurences in this document that markdown symbols are used inside of html. They should be replaced with `<code>` or `<a>` etc.